src/tester.rs

```diff
@@ -470,150 +484,163 @@ fn run_tests(
             }
 
             let tests = parse_tests(&test_str);
-
-            if !cfg!(unix)
-                && tests
-                    .values()
-                    .find(|t| t.status == Status::Signal)
-                    .is_some()
-            {
-                write_ignored(
-                    test_fname.as_str(),
-                    "signal termination not supported on this platform",
-                    inner,
-                );
+            if (inner.ignored && !tests.ignore) || (!inner.ignored && tests.ignore) {
+                write_ignored(test_fname.as_str(), "", inner);
```

Comment:
Was there a reason not to pass the ignore string from top of the test string here? I think it would be nice but I don't have a very strong opinion on this.

Reply:
It's just a bit annoying to pass it through :/

Reply:
I thought it might be something like that. That's fine it's not that important.